package/src/components/ChannelList/ChannelListMessenger.tsx

```diff
@@ -1,6 +1,5 @@
 import React, { useEffect, useState } from 'react';
-import { RefreshControl, StyleSheet, View } from 'react-native';
-import { FlatList } from 'react-native-gesture-handler';
+import { FlatList, RefreshControl, StyleSheet, View } from 'react-native';
```

Comment:
Can you leave a code comment here? Just to make sure someone doesn't switch it back to RNGH